server/services/team/service.ts

```diff
@@ -159,4 +167,40 @@ export class TeamService extends ServiceClass implements ITeamService {
 
 		return this.TeamMembersModel.findByTeamId(teamId).toArray();
 	}
+
+	async addMember({ _id, username }: IUser, userId: string, teamId: string): Promise<InsertOneWriteOpResult<ITeamMember> | null | undefined> {
```

Comment:
I'd avoid using the return of mongodb driver on the service as it can have complex objects that might not be able to be transferred via network. for this action I'd return a `boolean` or `ITeamMember`

Reply:
Done!